bin/gdb_wrapper

```diff
@@ -78,51 +79,64 @@ require 'ruby-debug-ide/attach/util'
 require 'ruby-debug-ide/attach/native_debugger'
 require 'ruby-debug-ide/attach/process_thread'
 
-debugger = choose_debugger(options.ruby_path, options.pid, options.gems_to_include, debugger_loader_path, argv)
+pids = get_child_pids(options.pid.to_s)
+attach_threads = Array.new
 
-trap('INT') do
-  unless debugger.exited?
-    $stderr.puts "backtraces for threads:\n\n"
-    process_threads = debugger.process_threads
-    if process_threads
-      process_threads.each do |thread|
-        $stderr.puts "#{thread.thread_info}\n#{thread.last_bt}\n\n"
-      end
-    end
-    debugger.exit
+pids.each_with_index do |pid, i|
+
+  if(i == 1)
+    argv = reset_port(ARGV) # Only the first process works with the initialized port, the rest need to be recalculated
   end
-  exit!
-end
 
-debugger.attach_to_process
-debugger.set_flags
+  attach_threads << Thread.new(argv) do |argv|
```

Comment:
Another possible way to do this is to replace `each` with `map` and then just chain call the result.